### ISSUE TITLE
Regarding fine tuning.

### ISSUE THREAD

**Issue by vsag** [metalbubble/CAM#20]
In the paper you say that you finetune the model, so do you finetune the whole network or only the part of the network where you introduced the 3*3*1024 convolutions followed by global average pooling followed by soft max. We have very less compute so this detail may help us wasting out time and help us concentrate on other tasks. 

Thank you.

**Comment by metalbubble** [comment]
Hi in the paper I finetuned the whole network, because it usually brings higher accuracy, compared to fine-tuning the last layer. But you can just fine-tune the last layer, you still will get the localization ability. See Sec.4 in the paper. In the experiments of that section, I just trained the linear SVMs on the localizable deep features, without fine-tuning the whole networks
